recipes/nose2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nose2" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "954a62cfb2d2ac06dad32995cbc822bf00cc11e20d543963515932fd4eff33fa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - nose2 = nose2:discover
+    - nose2-3.6 = nose2:discover
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - six >=1.7
+    - coverage >=4.4.1
```

Comment:
Is this a build dependency?

Reply:
@CJ-Wright , I am sorry, but how do I know if it is "run" or "build" dependency? Here, https://github.com/nose-devs/nose2/blob/master/requirements.txt, both "six" and "coverage" are listed as requirements, so I put them in the "build" section.

Reply:
`build` dependencies are packages needed at build time (usually because the software is compiled). Most pure python libraries only need `python` and `pip` to build the software.

Reply:
Thanks @CJ-Wright , that makes sense.